src/server/scripts/Spells/spell_item.cpp

```diff
@@ -2530,15 +2538,38 @@ class spell_item_unsated_craving : public SpellScriptLoader
         {
             PrepareAuraScript(spell_item_unsated_craving_AuraScript);
 
+            bool ExceptionMobs(Unit* target) const
```

Comment:
a better name would be `bool isException(Unit* target) const`

Reply:
sure